app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -290,3 +290,7 @@ div.form-group .pficon-info{
 a.btn-info span{
   padding-bottom: 4px;
 }
+
+.btn-toolbar .btn.btn-docs {
```

Comment:
Please combine this with the other `.btn-toolbar` override in line 146. 
Also no need for `.btn.btn-docs`, just either `.btn-docs` or `.btn`.